Formula/roapi.rb

```diff
@@ -0,0 +1,17 @@
+class Roapi < Formula
+  desc "Full-fledged APIs for static datasets without writing a single line of code"
+  homepage "https://roapi.github.io/docs"
+  url "https://github.com/roapi/roapi/archive/refs/tags/roapi-http-v0.5.2.tar.gz"
+  sha256 "d145f36063548b7d90baf03ab93bbd0b31775663a42a0eb13d4f7f217bf41df6"
+  license "Apache-2.0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args, "--bins", "roapi-http"
+  end
+
+  test do
+    assert_equal "roapi-http 0.5.2", shell_output("#{bin}/roapi-http -V").strip
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula